iree_tests/benchmarks/sdxl/benchmark_sdxl_rocm.py

```diff
@@ -181,6 +188,27 @@ def test_sdxl_rocm_benchmark(goldentime_rocm_e2e, goldentime_rocm_unet,
     job_summary_lines.append(mean_line)
     logging.getLogger().info(mean_line)
 
+    # unet compilation stats check
+    with open(f"{scheduled_unet_dir}/compilation_info.json", 'r') as file:
+        comp_stats = json.load(file)
+    unet_dispatch_count = int(comp_stats["stream-aggregate"]["execution"]["dispatch-count"])
+    assert unet_dispatch_count <= goldendispatch_rocm_unet, "SDXL scheduled unet dispatch count should not regress"
+    compilation_line = (f"Scheduled Unet Dispatch Count: {unet_dispatch_count}"
+                        f" (golden dispatch count {goldendispatch_rocm_unet})")
```

Comment:
You could format these results into a table

Current text:
```
SDXL Benchmark Summary:
E2E Benchmark Time: 1659.0 ms (golden time 1661.5 ms)

Scheduled Unet Benchmark Time: 449.0 ms (golden time 450.5 ms)
Scheduled Unet Dispatch Count: 1718 (golden dispatch count 1718)
Scheduled Unet Binary Size: 2088217 bytes (golden binary size 2088217 bytes)

Prompt Encoder Benchmark Time: 16.7 ms (golden time 19.0 ms)
Prompt Encoder Dispatch Count: 1571 (golden dispatch count 1571)
Prompt Encoder Binary Size: 785493 bytes (golden binary size 785493 bytes)

VAE Decode Benchmark Time: 288.0 ms (golden time 288.5 ms)
VAE Decode Dispatch Count: 250 (golden dispatch count 250)
VAE Decode Binary Size: 762067 bytes (golden binary size 762067 bytes)
```

Markdown table (using empty rows as dividers, could leave those off):

Metric | Current value | Expected/golden value
-- | -- | --
E2E Benchmark Time | 1659.0 ms |1661.5 ms
 | | 
Scheduled Unet Benchmark Time | 449.0 ms | 450.5 ms
Scheduled Unet Dispatch Count | 1718 | 1718
Scheduled Unet Binary Size | 2088217 bytes | 2088217 bytes
 | | 
Prompt Encoder Benchmark Time | 16.7 ms | time 19.0 ms
Prompt Encoder Dispatch Count | 1571 | 1571
Prompt Encoder Binary Size | 785493 bytes | 785493 bytes
 | | 
VAE Decode Benchmark Time | 288.0 ms | time 288.5 ms
VAE Decode Dispatch Count | 250 | 250
VAE Decode Binary Size | 762067 bytes | 762067 bytes

Markdown source:
```

Metric | Current value | Expected/golden value
-- | -- | --
E2E Benchmark Time | 1659.0 ms |1661.5 ms
 | | 
Scheduled Unet Benchmark Time | 449.0 ms | 450.5 ms
Scheduled Unet Dispatch Count | 1718 | 1718
Scheduled Unet Binary Size | 2088217 bytes | 2088217 bytes
 | | 
Prompt Encoder Benchmark Time | 16.7 ms | time 19.0 ms
Prompt Encoder Dispatch Count | 1571 | 1571
Prompt Encoder Binary Size | 785493 bytes | 785493 bytes
 | | 
VAE Decode Benchmark Time | 288.0 ms | time 288.5 ms
VAE Decode Dispatch Count | 250 | 250
VAE Decode Binary Size | 762067 bytes | 762067 bytes
```

---

Or could sort by metric:

Benchmark | Current time (ms) | Expected/golden time (ms)
-- | -- | --
E2E† | 1659.0 |1661.5
Scheduled Unet | 449.0 | 450.5
Prompt Encoder | 16.7 | 19.0
VAE Decode | 288.0 | 288.5

† E2E = Encode + Scheduled Unet * 3 + Decode

Benchmark | Current dispatch count | Expected/golden dispatch count
-- | -- | --
Scheduled Unet | 1718 | 1718
Prompt Encoder | 1571 | 1571
VAE Decode | 250 | 250


Benchmark | Current binary size (bytes) | Expected/golden binary size (bytes)
-- | -- | --
Scheduled Unet | 2088217 | 2088217
Prompt Encoder | 785493 | 785493
VAE Decode | 762067 | 762067

markdown source:
```
Benchmark | Current time (ms) | Expected/golden time (ms)
-- | -- | --
E2E† | 1659.0 |1661.5
Scheduled Unet | 449.0 | 450.5
Prompt Encoder | 16.7 | 19.0
VAE Decode | 288.0 | 288.5

† E2E = Encode + Scheduled Unet * 3 + Decode

Benchmark | Current dispatch count | Expected/golden dispatch count
-- | -- | --
Scheduled Unet | 1718 | 1718
Prompt Encoder | 1571 | 1571
VAE Decode | 250 | 250


Benchmark | Current binary size (bytes) | Expected/golden binary size (bytes)
-- | -- | --
Scheduled Unet | 2088217 | 2088217
Prompt Encoder | 785493 | 785493
VAE Decode | 762067 | 762067
```

I think I like the last version there

Reply:
This is a lot nicer, made the updates for markdown :)